docs/users-guide/expressions/isempty.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Isempty
+---
+
+# Isempty
+
+`isempty` checks if a value in a **string column** is an empty string (`""`).
+
+**In Metabase, you must combine `isempty` with another expression that accepts boolean values.** The table below shows you examples of the boolean output that will be passed to your other expression(s).
+
+| Syntax                                                             | Example with an empty string | Example with a true `null` |
+| ------------------------------------------------------------------ | ---------------------------- | -------------------------- |
+| `isempty(value)`                                                   | `isempty("")`                | `isempty(null)`            |
+| Returns `true` if the value is an empty string, `false` otherwise. | `true`                       | `false`                    |
+
+## How Metabase handles empty strings
+
+In Metabase, columns with string [data types][data-types] will display blank cells for empty strings _or_ `null` values (if the column is nullable in your database).
+
+For example, in the column below, the empty cells could contain either:
+
+- `""`: feedback was submitted and left intentionally blank, so the person had "no feedback to give".
+- `null`: no feedback was submitted, so the person's thoughts are "unknown".
+
+| Feedback           |
+| ------------------ |
+|                    |
+|                    |
+| I like your style. |
+
+## Replacing empty strings with another value
+
+| Feedback           | `case(isempty([Feedback]), "No feedback.", [Feedback])` |
+| ------------------ | ------------------------------------------------------- |
+|                    |                                                         |
+|                    | No feedback.                                            |
+| I like your style. | I like your style.                                      |
+
+Combine `isempty` with the [`case` expression](./case) to replace empty strings with something more descriptive.
+
+If the second row's blank cell is actually an empty string, then `isempty` will return `true`. The `case` statement evaluates `true` to return the first output "No feedback".
+
+If the first row's blank cell is actually a `null`, (or even an emoji that blends into your table background), then `isempty` will return `false`, and `case` will return whatever's in the Feedback column as the default output.
+
+## Accepted data types
+
+| [Data type][data-types] | Works with `isempty` |
```

Comment:
Do we need a table here?

Reply:
Gonna keep it, based on the assumption that people tend to scan these types of docs, so consistency is key.

If we make it easy to scan for "all green", it should be easier to catch people's attention when there are "reds" (hopefully it'll make them pause and read "Limitations" below).

Reply:
For posterity -- the hypothesis that came out of our discussion :)

> I’m also thinking about the case where people can’t tell the difference between the actual data type vs. the data types mapped to a string.
For example, if someone knows that a function “works with strings” and “doesn’t work with numbers”, a list of all of the data types can lead to the right question — “is my number somehow being typed as a string?”
Without that information, the person doesn’t have as much of a lead as to what’s going wrong.